src/Core/Services/Implementations/NotificationsApiPushNotificationService.cs

```diff
@@ -227,4 +228,23 @@ public Task SendPayloadToOrganizationAsync(string orgId, PushType type, object p
         // Noop
         return Task.FromResult(0);
     }
+
+    public async Task PushSyncOrganizationStatusAsync(Organization organization)
+    {
+        await PushOrganizationStatusAsync(organization, PushType.SyncOrganizationStatusChanged);
+    }
+
+    private async Task PushOrganizationStatusAsync(Organization organization, PushType type)
+    {
+        if (organization is not null)
+        {
+            var message = new OrganizationStatusPushNotification
+            {
+                OrganizationId = organization.Id,
+                Enabled = organization.Enabled
+            };
+
+            await SendMessageAsync(type, message, true);
```

Comment:
Last argument needs to be false, since we don't want to exclude current device.
```suggestion
            await SendMessageAsync(type, message, false);
```